MetaMask/metamask-mobile
# feat: add the abilty to hide the disconnect all button as well as showing and hiding the accounts row when necessary

## **Description**
Adds tracking to the user requesting network switching as well as his press of the confirmation button.


## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/3299

## **Manual testing steps**

1. Go to the in app browser and open have a connected dapp for example to ethereum main net
2. In the dapp select to change network, the modal will come up
3. Confirm that the modal looks like the screenshot below **After: WIthout Bugs**

## **Screenshots/Recordings**





| **Before: With Bugs**       | **After: WIthout Bugs**        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/e873ff5b-d2a1-4a8d-93da-6b5ca4c859bc"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/ab3c3304-4a3c-4423-9b2c-576a8b2f9b2d"> |





## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
